Framework/Algorithms/test/PhaseQuadMuonTest.h

```diff
@@ -103,6 +106,15 @@ class PhaseQuadMuonTest : public CxxTest::TestSuite {
     TS_ASSERT_DELTA(specImE[20], 0.0031, 0.0001);
     TS_ASSERT_DELTA(specImE[50], 0.0035, 0.0001);
   }
+  // add test for no phase
```

Comment:
Can we add these tests?

Reply:
I was concerned about having time to complete them for the release. I should get time later. If not they will be the next thing that I do